buildSrc/src/main/java/org/elasticsearch/gradle/test/rest/CopyRestApiTask.java

```diff
@@ -58,8 +58,10 @@
     final ListProperty<String> includeCore = getProject().getObjects().listProperty(String.class);
     final ListProperty<String> includeXpack = getProject().getObjects().listProperty(String.class);
     String sourceSetName;
+    boolean skipHasRestTestCheck;
```

Comment:
This is needed so that you can run compat tests for a project that may no longer have YAML REST tests in master. Also, there may be cases where a module is completely deleted in master, but we will still want to use this plugin to only test compatibility. 

This basically ensures that the execution of "compat" YAML tests is not directly coupled to the existence of "normal" YAML rest tests.